.github/workflows/build.yml

```diff
@@ -15,9 +15,8 @@
 name: Build
 on:
   push:
-    tags:
```

Comment:
this is redundant because the prerelease job does all the building and verification on tag push